moveit_ros/warehouse/warehouse/src/warehouse_services.cpp

```diff
@@ -133,11 +133,26 @@ int main(int argc, char** argv)
   int port;
   node.param<std::string>("warehouse_host", host, "localhost");
   node.param<int>("warehouse_port", port, 33829);
+  warehouse_ros::DatabaseConnection::Ptr conn;
 
-  warehouse_ros::DatabaseConnection::Ptr conn = moveit_warehouse::loadDatabase();
-  conn->setParams(host, port, 5.0);
+  try
+  {
+    conn = moveit_warehouse::loadDatabase();
+    conn->setParams(host, port, 5.0);
+
+    ROS_INFO("Connecting to warehouse on %s:%d", host.c_str(), port);
+    if (!conn->connect())
+    {
+      ROS_ERROR("Failed to connect to DB");
```

Comment:
It'd be nice to print the host and port on this error message. I'm surprised it ever worked without the call to **connect()**